collections/_drafts/agent-forwarding.md

```diff
@@ -251,7 +249,7 @@ Adding the `--verbose` flag shows the progress of starting the agent.
 The traditional way to run _gpg-agent_ on Unix systems is by launching it at
 login time.
 
-TO BE SURE, we will add the following line to
+To be sure, we will add the following line to&helip;
```

Comment:
```suggestion
To be sure, we will add the following line to&hellip;
```

Reply:
the discontinuity if this is very confusing and i am not yet sure what i was referring to